apps/api/src/admin/summary_handler.py

```diff
@@ -0,0 +1,26 @@
+from collections import Counter
+from typing import Union
+
+from pydantic import BaseModel, TypeAdapter
+
+from models.ApplicationData import Decision
+from services import mongodb_handler
+from services.mongodb_handler import Collection
+from utils.user_record import Role, Status
+
+
+class ApplicantSummaryRecord(BaseModel):
+    status: Union[Status, Decision]
```

Comment:
Nitpick: Could we extract `Union[Status, Decision]` as a type to reduce repetition?

Reply:
Refactored to `ApplicantStatus` in 8aa8dbc.